Update README and add quickstart (#113)
substratusai/substratus: README.md

# [Substratus.AI](https://www.substratus.ai)

Deploy and finetune Large Language Models on K8s in a breeze.

Main features:
* Deploy large models in minutes  (e.g. latest Large Language Models)
* Fine-tuning without writing code
* Built-in serving layer compatible with OpenAI APIs
* Run Notebooks to experiment

Learn more at [substratus.ai](https://www.substratus.ai).

## Intro and demo
Read the [Introduction](https://www.substratus.ai/docs/introduction) and
watch the video below:
[![Watch the video](https://img.youtube.com/vi/CLyXKJHIQ6A/hq2.jpg)](https://youtu.be/CLyXKJHIQ6A)

## Docs
Read the full docs at [substratus.ai/docs](https://www.substratus.ai/docs)

## Quickstart
Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)

## Architecture
To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.

## Developer docs

> See docs directory
> 
> 

Reviewer: ```suggestion

See the [development](docs/development.md) doc to get started.

```

> ## Disclaimer
> Substratus is still under active development and might introduce backward incompatible changes.

Reviewer: ```suggestion
## Disclaimer

Substratus is under active development and might introduce backward incompatible changes.
```

# [Substratus.AI](https://www.substratus.ai)

Deploy and finetune Large Language Models on K8s in a breeze.

Main features:
* Deploy large models in minutes  (e.g. latest Large Language Models)
* Fine-tuning without writing code
* Built-in serving layer compatible with OpenAI APIs
* Run Notebooks to experiment

Learn more at [substratus.ai](https://www.substratus.ai).

> ## Intro and demo
> Read the [Introduction](https://www.substratus.ai/docs/introduction) and
> watch the video below:
> [![Watch the video](https://img.youtube.com/vi/CLyXKJHIQ6A/hq2.jpg)](https://youtu.be/CLyXKJHIQ6A)
> 
> ## Docs
> Read the full docs at [substratus.ai/docs](https://www.substratus.ai/docs)
> 
> ## Quickstart
> Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)
> 
> ## Architecture
> To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.

Reviewer: ```suggestion
## Intro and demo

Read the [Introduction](https://www.substratus.ai/docs/introduction) and
watch the video below:
[![Watch the video](https://img.youtube.com/vi/CLyXKJHIQ6A/hq2.jpg)](https://youtu.be/CLyXKJHIQ6A)

## Docs

Read the full docs at [substratus.ai/docs](https://www.substratus.ai/docs)

## Quickstart

Read the [Quickstart Docs](https://www.substratus.ai/docs/quickstart)

## Architecture

To learn more about how Substratus works, check out the [Architecture](https://www.substratus.ai/docs/architecture) page.
```

# [Substratus.AI](https://www.substratus.ai)

> Deploy and finetune Large Language Models on K8s in a breeze.
> 
> Main features:
> * Deploy large models in minutes  (e.g. latest Large Language Models)

Reviewer: ```suggestion
Deploy and fine-tune Large Language Models on K8s in a breeze.

Features:

* Deploy large models in minutes  (e.g. latest Large Language Models)
```
